nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
I can see why convention over configuration is strong within a development community but what we observed across a wide variety of a user community was very different need.

Regarding what is arguably 'natural' from one language to another this is a great point that I personally had not considered.  Fortunately I believe the set of supported terms to accept as indicators of periods can be substantially expanded though I am not aware of how problematic ambiguity could be.  We could of course choose to support some specific standard for expression of time periods though so still have viable paths.